sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c

```diff
@@ -56,6 +56,22 @@ void gnrc_sixlowpan_nd_init(gnrc_ipv6_netif_t *iface)
     gnrc_ndp_internal_send_rtr_sol(iface->pid, NULL);
 }
 
+void gnrc_sixlowpan_nd_router_set_rtr_adv(gnrc_ipv6_netif_t *netif, bool enable)
+{
+    ipv6_addr_t all_routers = IPV6_ADDR_ALL_ROUTERS_LINK_LOCAL;
+
+    if (enable) {
+        netif->flags |= GNRC_IPV6_NETIF_FLAGS_RTR_ADV;
+        gnrc_ipv6_netif_add_addr(netif->pid, &all_routers, 128,
```

Comment:
Maybe you should at first check the return value of `gnrc_ipv6_netif_add_addr`, before you set the `netif->flags` in case there is no space left to add a new ip address..